### PR TITLE
GLEC ingress - Refrigerant gases and Fuel

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] New

**What category or categories does it affect?**

_Refrigerants and Fugitive Gases and Fuel_

**What?**

_ADD GLEC 2019 dataset_

**Why?**

_To add a key data set for the transport emissions calculations. _

### CODE REVIEW

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Ah few more things from the checker: There is a problem with the scope field - I think it is an autofill error - "1|2|14", uncertainty field should be empty, in case it is not supplied.

There are also several new values of lca_activity, also those I haven't tackled last time bc of our discussion on using the actual lca phases, rather than their unique combinations. Let's discuss it today again!

**Comment by patstirling** [comment]
> 

Note that you will need to delete the quotation marks as well if you are editing directly

**Comment by AnastasiaLobanova** [comment]
> > 
> 
> Note that you will need to delete the quotation marks as well if you are editing directly

They should go away automatically once we commit the change

**Review by AnastasiaLobanova — DISMISSED** [review]
looks good to me!

**Review by AnastasiaLobanova — APPROVED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -9599,6 +9774,7 @@ e0aa2ef4-5c4d-48fa-9c52-2753243805d5,Refrigerants and Fugitive Gases,Refrigerant
...
 fc3e562a-1007-4652-a740-12d44abcd9a7,Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_r_404a,R-404A,kg,not-supplied,3922,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,1|2|3,unknown,EPA,2021,2021,2007,US,,Climatiq,2022/01/22,Emission intensity of gas specified in 100-year GWPs from IPCC Fourth Assessment Report (AR4) 2007. GWPs of blended refrigerants are based on their HFC and PFC constituents. Retrieved from the EPA's GHG Emission Factors Hub (xlsx).,https://www.epa.gov/climateleadership/ghg-emission-factors-hub
+658be0c6-dacb-4e73-b942-82524267ce36,Refrigerants and Fugitive Gases,Refrigerants and Fugitive Gases,fugitive_gas-type_r_404a,R-404A - Mixture: 44.0% R-125 - 4.0% R-134a - 52.0% R-143a,kg,not-supplied,3921.6,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,1|2|3,unknown,GLEC,2019,2019 - 2022,,GLOBAL,,Cliimatiq,2022/06/20,Emission intensity of gas specified. Retrieved from the Global Logistics Emissions Council Framework for Logistics Emissions Accounting and Reporting. (2019).,https://www.smartfreightcentre.org/en/how-to-implement-items/what-is-glec-framework/58/
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:9777]: For this and other gases with "Mixture:" - do we need to be this explicit? E.g. is R-404A always this mix, or is this particular EF specific to this blend of R-404A? In the former case I'd argue we remove this entirely and the user can check the source if they way. For the latter, I'd argue we put this in the description - it seems too much detail for the name, unless it is a really important detail that helps the user distinguish this particular EF.

**Review by patstirling — CHANGES_REQUESTED** [review]
Great start! Few thoughts/things to tidy up

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -3811,6 +3817,12 @@ bfa28eb6-5d0e-473a-939c-af7e72502036,Energy,Fuel,fuel_type_anthracite_coal-fuel_
...
+482708c8-f454-466c-85fa-f21bdc28fce6,Energy,Fuel,fuel_type_aviation_fuel-fuel_use_na,Aviation fuel - TTW,kg,not-supplied,3.18,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,tank_to_wheel,GLEC,2019,2019 - 2022,,GLOBAL,m,Climatiq,2022/06/20,Tank-to-Wheel (TTW) emission intensity of fuel. Retrieved from the Global Logistics Emissions Council Framework for Logistics Emissions Accounting and Reporting. (2019). NOTE: The source does not clarify if the kgCO2e value is calculated using either IPCC Fourth Assessment Report (AR4) or IPCC Fifth Assessment Report (AR5) methodologies.,https://www.smartfreightcentre.org/en/how-to-implement-items/what-is-glec-framework/58/
```
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:3820]: so both the fuel use and fuel type should be aviation?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -3796,6 +3796,12 @@ defcf49b-ba06-4420-bebd-e87778696395,Energy,Energy Services,energy_services-type
...
+eb24f71f-467f-452a-bed9-5542bfc5d48a,Energy,Fuel,fuel_type_100_percent_biodiesel-fuel_use_na,100% biodiesel (B100) - TTW,kg,not-supplied,0,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,,tank_to_wheel,GLEC,2019,2019 - 2022,,EU,m,Climatiq,2022/06/20,Tank-to-Wheel (TTW) emission intensity of fuel. Retrieved from the Global Logistics Emissions Council Framework for Logistics Emissions Accounting and Reporting. (2019). NOTE: The source does not clarify if the kgCO2e value is calculated using either IPCC Fourth Assessment Report (AR4) or IPCC Fifth Assessment Report (AR5) methodologies.,https://www.smartfreightcentre.org/en/how-to-implement-items/what-is-glec-framework/58/
```
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:3799]: there is no scope mentioned in GLEC. I had not-supplied in the field first but due to the schema error changed it to blank

File: OpenEmissionFactorsDB.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:3820]: so both the fuel type and fuel use should be aviation?

[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:3820]: yes, this makes "fuel_use" as a sub-field queryable; the key for machine readability is to remain predictable in the location of data (even if to a human it feels redundant to have the information twice)

File: OpenEmissionFactorsDB.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:3799]: so is this a bug in the schema?

**Review by patstirling — APPROVED** [review]

**Pull request merged**